resources/design-document.md

```diff
@@ -146,6 +156,14 @@ String contestStatus;
 * Accepts `GET` requests to `/groups`
 * Retrieves all groups
 
+## 5.8 Get All Games Endpoint
+* Accepts `GET` requests to `/games`
+* Retrieves all games
+
+## 5.9 Get User Stats Endpoint
+* Accepts `GET` requests to `/games`
```

Comment:
Is this intended to a different path than `/games`? (maybe something like `/statistics`?)